elasticsearch/initialize_es_index.py

```diff
@@ -3,39 +3,58 @@
 1. es_index_prefix (CLI parameters)
```

Comment:
I think that this script should go to the scripts folder, these folder for ES mappings and settings only. It's good practice to have all scripts in one place.

Reply:
done in bf3d732